curator/actions.py

```diff
@@ -1307,28 +1307,64 @@ def _get_reindex_args(self, source, dest):
             del reindex_args['slices']
         return reindex_args
 
-    def _post_run_quick_check(self, index_name):
-        # Verify the destination index is there after the fact
-        index_exists = self.client.indices.exists(index=index_name)
-        alias_instead = self.client.indices.exists_alias(name=index_name)
-        if not index_exists and not alias_instead:
-            self.loggit.error(
-                'The index described as "{0}" was not found after the reindex '
-                'operation. Check Elasticsearch logs for more '
-                'information.'.format(index_name)
+    def get_processed_items(self, task_id):
+        """
+        This function calls client.tasks.get with the provided `task_id`.  It will get the value 
+        from ``'response.total'`` as the total number of elements processed during reindexing.
+        If the value is not found, it will return -1
+
+        :arg task_id: A task_id which ostensibly matches a task searchable in the
+            tasks API.
+        """
+        try:
+            task_data = self.client.tasks.get(task_id=task_id)
+        except Exception as e:
```

Comment:
Lines 1321 and 1322 are not tested. A mock exception raised in a unit test would satisfy this.